sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulseDataFetcherTests.java

```diff
@@ -89,8 +95,10 @@ void endpointChangesWithRedirectHeaderAndGetNewPingInterval() {
         headers.put("x-ms-qps-service-polling-interval-hint", "1000");
         headers.put("x-ms-qps-service-endpoint-redirect-v2", "https://new.endpoint.com");
         headers.put("x-ms-qps-subscribed", "true");
+        headers.put("x-ms-qps-configuration-etag", "0::randometag::1::");
```

Comment:
nit: since this test is testing pings, I would set the etag to empty string

Reply:
fixed